Automattic/sensei
# Update the WP Coding Standards

## Proposed Changes
* Update `wp-coding-standards/wpcs` to 3.0.1 following [this guide](https://github.com/WordPress/WordPress-Coding-Standards/wiki/Upgrade-Guide-to-WordPressCS-3.0.0-for-ruleset-maintainers).
* Update the coding standard rules using [this script ](https://code.a8c.com/source/wpcom-git/browse/trunk/bin/phpcs-update-rules.sh?view=raw).
* Make fixes for a new rule - `WordPress.Security.EscapeOutput.ExceptionNotEscaped`.

## Testing Instructions


1. CI should be passing.
2. Run `composer install` and make sure the plugin is working as before.

## Pre-Merge Checklist
